plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/EclipseProjectCreator.java

```diff
@@ -15,10 +25,57 @@
   /**
    * @return status of project creation
    */
-  static IStatus makeNewProject(AppEngineStandardProjectConfig config, IProgressMonitor monitor) {
-    IWorkspaceRoot workspaceRoot = ResourcesPlugin.getWorkspace().getRoot();
-    IProject project = workspaceRoot.getProject(config.getEclipseProjectName());
-    return Status.OK_STATUS;
+  static IStatus makeNewProject(
+      AppEngineStandardProjectConfig config, IProgressMonitor monitor,
+      final Shell shell, final IWizardContainer container) {
+    
+    // todo submonitor????
+    
+
+    URI location = null;
+    if (!config.getUseDefaultProjectLocation()) {
+      location = config.getEclipseProjectLocationUri();
+    }
```

Comment:
I think this if-default-then-use-this-directory should be decided in the wizard page, no?